### DISCUSSION TITLE
NL regional list

### DISCUSSION THREAD

**Discussion by Procyon-b** [uBlockOrigin/uBlock-issues#3250] [General] [answered]
The name has been changed to "AdGuard Dutch" but the content is still "EasyDutch" when I view the content. (even after clicking the clock). Is the problem on my side ? Note: "home" icon links to AdGuard.

Chrome uBO-1.57.3rc1

**Comment by gorhill** [comment]
> content is still "EasyDutch"

Probably because it has not updated yet.

**Reply by Procyon-b** [reply]
@gorhill
An update to tell you that, despite the name change, the list is still "_EasyDutch_"
In 1.58.1b2 the automatic update is still at least 1 day away. (I had disabled/enabled the list before starting this discussion).
I had 1.58.0 disabled, and enabled it a few minutes ago. I immediately opened the "_Filter lists_" panel and witnessed this:

- The name was still "Easylist ..."
- I saw the clock turn blue
- Moments later I saw the name changed to "_AdGuard Dutch"_
- Right now, the clock bubble says "Last update: 10 minutes ago"
- Filters count: "2,743 used out of 2,772"
- If I click the eye to open the list: 
```
! Title: EasyDutch
! Last updated: Tue, 16 Jan 2024 13:20:57 +0000
! Expires: 7 days (update frequency)
! Diff-Path: patches/2024.1.16.800.patch#easydutch
! Diff-Expires: 6 hours
```
Note: it's the same header in 1.58.1b2

**Reply by gorhill** [reply, marked as answer]
Ok, I think it's because of lingering `cdnURLs` property which was present for the previous list. I need to set that property to `null` for uBO to forcefully remove that property.

https://github.com/gorhill/uBlock/commit/cc417a0cd48067c117d48fdd648c5d356d892c84

**Reply by Procyon-b** [reply]
I can confrim that the list is correct now (1.58.1b2)

**Comment by notDavid** [comment]
Hi there, does anyone know *why* EasyDutch was replaced with AdGuard Dutch ?

Just curious, and the related discussion mentioned in https://github.com/gorhill/uBlock/commit/f1f63c940b13411a91f063ce429518cf3cd29b23 has disappeared.

Thank you!

**Reply by stephenhawk8054** [reply]
It's an internal discussion. We couldn't contact with the maintainers at that time to fix the issues: https://github.com/uBlockOrigin/uAssets/issues/23325#issuecomment-2078202417 so we switched to AG Dutch.

**Reply by Ignotum77** [reply]
@notDavid and @stephenhawk8054 Sadly, but true, life made an unexpected turn. This caused the maintainers to not be able to respond. After 2 years I am back. However I lost my credentials of @Nomes77 and thus of EasyDutch. This caused me to make a new account and org (https://github.com/EasyDutch-uBlockOrigin/EasyDutch) which is currently maintained.
